### PR TITLE
Remove formatting instructions.

### PR DESCRIPTION
**Pull request by ericsciple**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -176,9 +175,9 @@ gulp.task('test', ['testResources'], function () {
     shell.mkdir('-p', _testTemp);
 
     var suitePath = path.join(_testRoot, options.suite + '/_suite.js');
-
+    var tfBuild = ('' + process.env['TF_BUILD']).toLowerCase() == 'true'
```
Comment (ericsciple) [inline gulpfile.js]: The CI flag doesn't *need* to be removed, I just thought it might be neat to leverage our TF_BUILD variable instead :)

Comment (ericsciple) [inline gulpfile.js]: Otherwise the definitions simply need to be updated.

Comment (bryanmacfarlane) [inline gulpfile.js]: I wonder if TF_BUILD is set in xplat agent.  Make sure it's in vsts-agent

**Comment by bryanmacfarlane** [comment]
Looks good.  Make sure we get into 99 release branch as well.  Bump VSO to 99 release pkg

**Comment by ericsciple** [comment]
@bryanmacfarlane Yikes, I didn't realize this was reviewed 6 days ago. Still want this in 99 or better to wait at this point?

**Comment by bryanmacfarlane** [comment]
Let's get it in.  Have Scott update

**Pull request merged**